postgresql/spoon.me

```diff
@@ -0,0 +1,54 @@
+##
+# Licensed under the Apache License, Version 2.0
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Setup meta
+meta title="postgresql"
+meta namespace="postgresql" 
+meta name="postgresql"
+
+#Basic dependencies and tools
+from clean,microsoft/vcredist:2008
```

Comment:
ok

I build this image but it fails to run on my machine. It also needs some
more work as we have to run postgres as service and not from startup-file.
It is needed so it can be safely mixed with other images during layering

I will do this and push updates directly to repo

2015-06-26 10:29 GMT+02:00 Roman Stoffel <notifications@github.com>:

> In postgresql/spoon.me
> <https://github.com/spoonapps/spoonme/pull/47#discussion_r33337392>:
>
> > @@ -0,0 +1,54 @@
> > +##
> > +# Licensed under the Apache License, Version 2.0
> > +# http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +# Setup meta
> > +meta title="postgresql"
> > +meta namespace="postgresql"
> > +meta name="postgresql"
> > +
> > +#Basic dependencies and tools
> > +from clean,microsoft/vcredist:2008
>
> No...postgre sql explicitily need it. It states in the documentation that
> you should install it when using the Zip installer!!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spoonapps/spoonme/pull/47/files#r33337392>.
>



-- 
Regards
Piotr Galiszewski